tools/ScenarioTest.ResourceManager/Assert.ps1

```diff
@@ -67,7 +67,7 @@ function Assert-ThrowsContains
   }
   catch
   {
-    if ($message -ne "")
+    if ($compare -ne "")
```

Comment:
Good job, thanks for correction. Could you help to correct Line108 as well?

Reply:
pending